src/Glpi/Error/ErrorHandler.php

```diff
@@ -223,19 +220,7 @@ public static function displayCaughtExceptionMessage(\Throwable $exception): voi
      */
     private function configureErrorReporting(): void
     {
-        $reporting_level = E_ALL;
-        foreach (self::ERROR_LEVEL_MAP as $value => $log_level) {
-            if (
-                \in_array($log_level, [LogLevel::DEBUG, LogLevel::INFO, LogLevel::NOTICE], true)
-                && !\in_array($this->env, [GLPI::ENV_DEVELOPMENT, GLPI::ENV_TESTING], true)
-            ) {
-                // Do not report debug, info, and notice messages unless in development/testing env.
-                // Notices are errors with no functional impact, so we do not want people to report them as issues.
-                // Suppressing the INFO level will prevent deprecations to be pushed in other environments logs.
-                $reporting_level &= ~$value;
-            }
-        }
-        \error_reporting($reporting_level);
+        \error_reporting(E_ALL);
```

Comment:
Not sure about this change; won't this report too much?

Reply:
We were wondering with @cedric-anne what was the point of this code since we already have a specific log level per env.

After some testing it need to be restored because it is used to prevent deprecated messages from being shown in production when the debug mode is active.

I'll document it so we remember it for next time.